fbchat/client.py

```diff
@@ -2106,6 +2106,15 @@ def getThreadIdAndThreadType(msg_metadata):
                     delta = m["delta"]
                     delta_type = delta.get("type")
                     delta_class = delta.get("class")
+                    
+                    delta_attachments = delta.get('attachments')
+                    lwa_type = None
+
+                    for a in delta_attachments:
```

Comment:
Going through attachments is already done in [listening for messages](https://github.com/carpedm20/fbchat/blob/3e7b20c379d552944046ab2f3224b9e7903c9d7f/fbchat/client.py#L2386)